packages/contentful/migrations/crn/researchOutputs/20230724090200-unique-urls.js

```diff
@@ -0,0 +1,17 @@
+module.exports.description = 'Remove uniqueness constraint on link field';
```

Comment:
do we really want to remove that? I wonder because even in the frontend we have the url constraint

![Screenshot 2023-07-27 at 08 09 26](https://github.com/yldio/asap-hub/assets/16595804/ab576ddd-4fde-423a-a4b5-3907ab6cd1ba)

so I wonder if we don't want to enforce that in Contentful as well

But even it is only for the migration because in old times we didn't have this constraint and people were able to create duplicate links in prod, maybe it would be nice to create a ticket to add this constraint back once the migration is done?

Reply:
If we enforce in contentful then _a lot_ of existing data from prod won't migrate. 

It was there originally, but there's a lot of data in Squidex prod with non-unique links, so I removed it.

Reply:
hmm ok, and if we add this back after the migration is done if people edit these research outputs with duplicate links they won't be able to publish. Yeah, it makes sense to remove it :)

Reply:
That's how it currently is in Squidex, so I doubt it would have _significant_ impact to add it back post-migration.